src/module/actor/lancer-actor.ts

```diff
@@ -91,39 +95,67 @@ interface BoundedValue {
   value: number;
 }
 
-/**
- * Extend the Actor class for Lancer Actors.
- */
-export class LancerActor<T extends LancerActorType> extends Actor {
-  data!: FoundryRegActorData<T> & {
-    // TODO: update to match template
-    data: {
-      // Temporary action store.
-      actions?: ActionData;
-
-      // Include additional derived info
-      derived: {
-        // These are all derived and populated by MM
-        current_hp: { max: number, value: number }; // -hps are useful for structure macros
-        current_heat: BoundedValue;
-        current_stress: BoundedValue;
-        current_structure: BoundedValue;
-        current_repairs: BoundedValue;
-        overshield: BoundedValue; // Though not truly a bounded value, useful to have it as such for bars etc
-
-        // Other values we particularly appreciate having cached
-        evasion: number;
-        edef: number;
-        save_target: number;
-        speed: number;
-        armor: number;
-        // todo - bonuses and stuff. How to allow for accuracy?
-      };
-    };
+interface DerivedProperties<T extends EntryType> {
+  // These are all derived and populated by MM
```

Comment:
Are these populated by MM specifically, or by code we use that reads from mm? If the former, is there an MM type we could use instead of listing them out?

Reply:
This was copied (including the comment) from within the previous verson of  Actor.  @whitespine would probably have a better idea.

Reply:
Gotcha. We can leave it be for now then probably.